### ISSUE TITLE
Support for Fody v5

### ISSUE THREAD

**Issue by Examed** [csnemes/tracer#82]
Thank you for the creation of Fody.Tracer I really like it.

The only annoying thing is the limitation of the Fody version.
It would be cool if you add support for Fody v5 so I get rid of the reference problems.

**Comment by csnemes** [comment]
Ok, I'll update it to v5 and create a new package.

**Comment by csnemes** [comment]
I only updated to v4 as I think v5 is too restrictive, but removed the cap from nuget so one can upgrade it to v5.

**Comment by Examed** [comment]
thanks

**Comment by Examed** [comment]
ohh man I'm sorry to borther you again but NullGuard.Fody 2.0 needs a minimum of Version 5.0.6.
Its not that urgent so it could just be adjusted in the next update :) 

Thanks

**Comment by Examed** [comment]
Hi its me again would it be possible to push the upper limit above 6.0.0 or maybe even remove the upper limit completely like it its done at NullGuard.Fody (for example) 

Thanks :)

**Comment by frankHarmann** [comment]
Yes, it would be great if you could push the upper limit to 6.x or remove it completly.
Thanks

**Comment by csnemes** [comment]
Removed the cap as temp solution. I try to delay the upgrade fody in tracer as fody 5.x does not support vs2017.
